learn/intro/obol-splits.md

```diff
@@ -24,18 +24,28 @@ An Obol Validator Manager (OVM) is a smart contract which manages the deposit, w
+* `principalRecipient` - This is the **address where the principal will be returned** to when validators exit or a withdrawal above the `principalThreshold` is made. This can be changed later by the `owner` or addresses with the `SET_PRINCIPAL_ROLE`.
+* `rewardRecipient` - This is the **address where the accrued ether reward will be sent** when `distributeFunds()` is called. Usually it is a [Pull Split](https://docs.splits.org/core/split-v2#how-it-works) from [splits.org](https://splits.org). This can be changed later by the `owner` or addresses with the `SET_REWARD_ROLE`.
+* `principalThreshold` - This is a configurable amount of Ether which dictates at what amount of value in the contract should we consider it to be principal being returned rather than reward accrued. The amount is immutable. A sensible default here is 16 ether (16000000000 gwei), the threshold used in Obol's earlier [Optimistic Withdrawal Recipients](obol-splits.md#optimistic-withdrawal-recipient​httpsdocsobolorglearnintroobol-splitsoptimistic-withdrawal-recipient-a-hrefwithdrawal-recipients-idwithdrawal-recipientsa-a-hrefoptimistic-withdrawal-recipient-idoptimistic-withdrawal-recipienta). Further detail in the [FAQ](obol-splits.md#faq) section.
 
 ### Roles
 
-Obol Validator Managers implement standard Role-Based Access Control. The OVM has the following roles that can be granted by the OVM owner.
+Obol Validator Managers implement standard Role-Based Access Control. The OVM has the following roles that can be granted by the OVM owner, using the `grantRoles()` function.
 
 * `WITHDRAWAL_ROLE`: Permits an address to trigger a partial withdrawal, or full exit of all validators managed by this contract using [EIP7002](https://eips.ethereum.org/EIPS/eip-7002).
 * `CONSOLIDATION_ROLE`: Permits an address to initiate a consolidation between one or more source validators and a target validator, all managed by this contract. All source and target validators must be active with a balance greater than 32 ether.
-* `SET_PRINCIPAL_ROLE`: Permits an address to change the recipient of the principal returned when validators exit, or a withdrawal above the principalThreshold is initiated.
+* `SET_PRINCIPAL_ROLE`: Permits an address to change the recipient of the principal returned when validators exit, or a withdrawal above the principalThreshold is initiated. Also this permits an address to adjust the amount of principal stake being tracked by the contract.
 * `RECOVER_FUNDS_ROLE`: Permits an address to initiate `ERC20.transfer()` calls to arbitrary external addresses, with the intent to recover otherwise stuck tokens.
+* `SET_REWARD_ROLE`: Permits an address to change the recipient of the reward when `distributeFunds()` is called.
+* `DEPOSIT_ROLE`: Permits an address to call the `deposit()` function.
```

Comment:
this is super nitpicky, but should we alphabetise or logically order these roles? e.g. withdrawal first, deposit last seems weird. 
Logical order: deposit, consolidate, withdraw, set principal, set rewards, recover funds. 
alphabetic order: consolidate, deposit, set principal, set rewards, recover funds, withdraw. 

i think i favour the 'logical' ordering

Reply:
Perfect point, updated to match the suggested logical ordering.